### PR TITLE
DEV: Updated override_on to override status and changed the Signal to…

### PR DESCRIPTION
**Pull request by slacAdpai**
… EpicsSignalRO. Also updated open_override to override_force_open.

## Description
Updated override names and changed override_on/Override status to be readback only.

### CODE REVIEW

File: pcdsdevices/valve.py [hunk 1]
```diff
@@ -180,11 +180,11 @@ class ValveBase(Device):
 
 class VVC(ValveBase):
     """Vent Valve, Controlled."""
-    override_on = Cpt(EpicsSignalWithRBV, ':OVRD_ON', kind='omitted',
-                      doc='Epics Command to set/reset Override mode')
-    open_override = Cpt(EpicsSignalWithRBV, ':FORCE_OPN', kind='omitted',
-                        doc=('Epics Command for open the valve in override '
-                             'mode'))
+    override_status = Cpt(EpicsSignalRO, ':OVRD_ON', kind='omitted',
```
Comment (ZryletTC) [inline pcdsdevices/valve.py:183]:
```suggestion
    override_status = Cpt(EpicsSignalRO, ':OVRD_ON_RBV', kind='omitted',
```

File: pcdsdevices/valve.py [hunk 2]
```diff
@@ -180,11 +180,11 @@ class ValveBase(Device):
...
+                          doc='Epics Readback on Override mode')
+    override_force_open = Cpt(EpicsSignalWithRBV, ':FORCE_OPN', kind='omitted',
+                              doc=('Epics Command for open the valve in '
```
Comment (ZryletTC) [inline pcdsdevices/valve.py:186]:
```suggestion
                              doc=('Epics Command to force open the valve in'
```

**Comment by slacAdpai** [comment]
Tyler, Thank you Thank you!

**Comment by ZryletTC** [comment]
No problem. @klauer can you merge?

**Pull request merged**